### ISSUE TITLE
the evaluation code

### ISSUE THREAD

**Issue by lijiaxinxin** [ecnuycxie/DG-Font#33]
Would you share the eval code?
The specific implement of the evaluation is not very clear.
I am looking forward to knowing the details of how to cauculate the SSIM FID LPIPS RMSE and L1 loss.
I also wonder how you calculate these metrics? Did you convert the 190 characters in the test set from Song to 399 other styles, and then did the calculation? Or use other methods?
Thanks

**Comment by ecnuycxie** [comment]
yes, those 190 characters are converted to 399 different styles. We calculate the SSIM LPIPS RMSE L1 loss using generated images and groud truth. Besides, we calculate the fid between all 990 images and 190 generated images.

**Comment by jingmingtao** [comment]
请问效果能复现出来吗
